tool/format_coverage.dart

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This is an alternative front end to package:coverage (to better support merging results from multiple runs)? If so, perhaps add a file or class comment to that effect.

Reply:
this is just a temporary fork as I mentioned above.  with the next release of coverage, i will constrain version and eliminate the fork.

Reply:
gotcha